src/udev/cdrom_id/cdrom_id.c

```diff
@@ -308,17 +308,16 @@ static int cd_media_compat(Context *c) {
 
         r = ioctl(c->fd, CDROM_DRIVE_STATUS, CDSL_CURRENT);
         if (r < 0)
-                return log_debug_errno(errno, "ioctl(CDROM_DRIVE_STATUS) failed: m");
-        if (r != CDS_DISC_OK) {
-                log_debug("ioctl(CDROM_DRIVE_STATUS) → %d (%s), ignoring",
-                          r,
-                          r == CDS_NO_INFO ? "no info" :
-                          r == CDS_NO_DISC ? "no disc" :
-                          r == CDS_TRAY_OPEN ? "tray open" :
-                          r == CDS_DRIVE_NOT_READY ? "drive not ready" :
-                          "unkown status");
-                return -ENOMEDIUM;
-        }
+                return log_debug_errno(errno, "ioctl(CDROM_DRIVE_STATUS) failed: %m");
+        if (r != CDS_DISC_OK)
+                log_debug_errno(SYNTHETIC_ERRNO(ENOMEDIUM),
+                                "ioctl(CDROM_DRIVE_STATUS) → %d (%s), ignoring.",
+                                r,
+                                r == CDS_NO_INFO ? "no info" :
+                                r == CDS_NO_DISC ? "no disc" :
+                                r == CDS_TRAY_OPEN ? "tray open" :
+                                r == CDS_DRIVE_NOT_READY ? "drive not ready" :
+                                "unkown status");
```

Comment:
Typo in unk**n**own and the return was lost!

Reply:
This was … embarrassing. Fixup in https://github.com/systemd/systemd/pull/19355.